addons/stock/wizard/stock_quant_relocate.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-is_not_full_package
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+from ast import literal_eval
+
+class RelocateStockQuant(models.TransientModel):
+    _name = 'stock.quant.relocate'
+    _description = 'Stock Quantity Relocation'
+
+    quant_ids = fields.Many2many('stock.quant')
+    company_id = fields.Many2one(related="quant_ids.company_id")
+    dest_location_id = fields.Many2one('stock.location', domain="[('usage', '=', 'internal'), ('company_id', '=', company_id)]")
+    dest_package_id_domain = fields.Char(compute="_compute_dest_package_id_domain")
+    dest_package_id = fields.Many2one('stock.quant.package', domain="dest_package_id_domain", compute="_compute_dest_package_id", store=True)
+    message = fields.Text('Reason for relocation')
+    is_not_full_package = fields.Boolean(compute='_compute_is_not_full_package')
+    partial_package_names = fields.Char(compute="_compute_is_not_full_package")
+    is_not_single_location = fields.Boolean(compute='_compute_is_not_single_location')
+
+    @api.depends('quant_ids')
+    def _compute_is_not_full_package(self):
+        for wizard in self:
+            wizard.is_not_full_package = False
+            wizard.partial_package_names = ''
+            packages = wizard.quant_ids.package_id
+            incomplete_packages = packages.filtered(lambda p: any(q not in wizard.quant_ids.ids for q in p.quant_ids.ids))
+            if packages and incomplete_packages:
+                wizard.is_not_full_package = True
+                wizard.partial_package_names = ', '.join(n for n in incomplete_packages.mapped('display_name')[::-1])
```

Comment:
when I was testing, I was using package names like test, test2, test3, etc... and they would appear in reverse. So I used that to have them in order. Which is kind of dumb because they're ordered by id and not name. My bad